src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -27,14 +27,6 @@
 # pylint: disable=too-many-statements
```

Comment:
As taking thew new API from the managed cluster operation group, the previous container service client factory (on line 6) should be removed.

> src/azure-cli/azure/cli/command_modules/acs/commands.py:6:0: W0611: Unused cf_container_services imported from azure.cli.command_modules.acs._client_factory (unused-import)